DNN Platform/Website/DesktopModules/Admin/ViewProfile/Settings.ascx

```diff
@@ -7,7 +7,7 @@
 	</div>
     <div class="dnnFormItem">
         <dnn:Label ID="plTemplate" runat="server" ControlName="txtTemplate" />
-        <asp:TextBox ID="txtTemplate" Columns="60" TextMode="MultiLine" Rows="25" MaxLength="2000" runat="server" Width="300px" />
+        <asp:TextBox ID="txtTemplate" Columns="60" TextMode="MultiLine" Rows="25" runat="server" Width="300px" />
```

Comment:
If this change should happen, can we separate it out into its own PR?  Thanks!
```suggestion
        <asp:TextBox ID="txtTemplate" Columns="60" TextMode="MultiLine" Rows="25" MaxLength="2000" runat="server" Width="300px" />
```

Reply:
Sorry, I was a bit confused...